en/docs/observe/api-manager-analytics/configure-analytics/configure-alerts.md

```diff
@@ -0,0 +1,131 @@
+# Working with Alerts
+![alert config page]({{base_path}}/assets/img/observe/alerts/alert-config-full.png)
+
+One of the most useful features in APIM cloud analytics is the **Alerting support**. New analytics solution allows
+ your configure several types of alert and then receive alerts based on the configurations, either within
+  the portal or as emails. These alerts allow you to proactively monitor your API ecosystem and take corrective
+   meassures for any abnormalities you find. 
+   
+Users can configure alerts for each environment they have separately withing their organizations. Alert
+ configurations will always be per API basis. Users can also modify/remove already added alerting configurations. 
+ 
+Alert issuance is subjected to a suppression policy so that users will not be overwhelmed with duplicate alert
+ notifications. For each alert rule that you configure, there will be a 10 minute suppression window applied by
+  default. Suppression interval is currently not configurable. Also, please note that suppression will work on a per
+   alert rule basis. As the document progresses we will talk about different alert types and how to configure them.
+    
+## Alert Types
+### Latency Alert 
+Latency alerts allows users to configure alerts if response latency of their APIs are greater than a predefined
```

Comment:
Configure Latency alerts to identify if the response latency of APIs is greater than a predefined threshold. This is useful when you have APIs that should honor SLAs and when you want to know about slow APIs proactively. 

The following topics explain how to create, modify, and remove latency alerts.

Reply:
Fixed with 48dd54c